cli/js/compiler.ts

```diff
@@ -1349,9 +1205,12 @@ async function compile(
     }
   }
 
+  const end = new Date();
+  // @ts-ignore
+  util.log("time in internal TS", end - start);
```

Comment:
Just don't reply on implicit coersion:

```suggestion
  util.log("time in internal TS", Number(end) - Number(start));
```